srcpkgs/ananicy/files/ananicy/run

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+printf '0 \n' > /proc/sys/kernel/sched_autogroup_enabled
```

Comment:
I think we should leave this to the user and document it.

Reply:
I think that is a bad idea, because it leads to a service that does not really work ootb. The script currently follows what the [upstream systemd service](https://github.com/Nefelim4ag/Ananicy/blob/master/ananicy.service) does, but uses printf instead of sysctl as the later is not exactly suited for use in chainloading...

I cooperated with @linuxer in order to create and test an [ananicy service](https://github.com/mobinmob/void-66-services/blob/devel/usr/share/66/service/ananicy) for void-66-services. In an execline script sysctl is used in an if block. In a posix shell script, printf seems like a good choice.